README.md

```diff
@@ -45,6 +45,8 @@ We all know BitBucket lacks some features that we have in other platforms like G
     of the template in the options page, in case you don't want to include
     the file in your repository, e.g., https://gist.githubusercontent.com/anonymous/8054a3ee32f7cf1a5975e3fd52b3c5f3/raw/f6897720e8b6b93becd246187dac36038291c3a4/PULL_REQUEST_TEMPLATE.md.
 
+                  	Note: Currently, externally hosted pull request templates outside of BitBucket's domain is not supported with the Firefox addon.
+

```

Comment:
This is showing up as a code block because of the amount of tabs, insert just one tab instead of two

Before:
![image](https://user-images.githubusercontent.com/7514993/38589274-1178a8ae-3cf8-11e8-997a-5de7146af51a.png)

After:
![image](https://user-images.githubusercontent.com/7514993/38589283-1fb37a3e-3cf8-11e8-88a9-99c996f1105b.png)

Here's the text in case you want to copy it:

```markdown

	<em>Note: Currently, externally hosted pull request templates outside of BitBucket's domain is not supported with the Firefox addon.</em>

```

Reply:
Hahaha...i just realigned it before you commented. Take a peek.